### ISSUE TITLE
What is args ?

### ISSUE THREAD

**Issue by ilhamsj** [express-rate-limit/rate-limit-redis#202]
### Description

Some people may not understand this part.
`sendCommand: (...args: string[]) => client.sendCommand(args),`

### Why

some people may not understand args part.

### Alternatives

please add more detail description can use `rate-limit-mongo` for reference

**Comment by nfriedly** [comment]
Yeah, that's a fair point. `args` is an array containing the command and any values that rate-limit-redis passes to the database driver.

I'd like to put out an updated version of this library that has a default redis driver built-in, so that the whole `sendCommand` thing is unnecessary for basic usage, which should reduce that confusion.

**Comment by moohbr** [comment]
@nfriedly did want some help to do it? I have some free time until my colleague starts.

**Comment by nfriedly** [comment]
Sure. I just filed #203 with my thinking. I'd be interested in hearing if @wyattjoh or @gamemaker1 had any thoughts on the idea also.
